### PR TITLE
Mrtk v next input action rollback2

### PR DESCRIPTION
**Pull request by StephenHodgson**
Rolling back the interface implementation to fix issues with profiles.

### CODE REVIEW

File: Assets/MixedRealityToolkit-Tests/InteractionDefinitionTests.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ public class InteractionDefinitionTests
         [Test]
         public void Test01_TestObjectChanged()
         {
-            var interaction = new MixedRealityInteractionMapping(1, AxisType.Raw, DeviceInputType.None, (InputAction)InputAction.None);
+            var interaction = new MixedRealityInteractionMapping(1, AxisType.Raw, DeviceInputType.None, (MixedRealityInputAction)MixedRealityInputAction.None);
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit-Tests/InteractionDefinitionTests.cs]: Need to change this.

**Comment by keveleigh** [comment]
What's the benefit to removing the interface? Or is this a temporary rollback?

**Comment by StephenHodgson** [comment]
It broke the profiles by making them an interface. This fixes that issue.

**Comment by SimonDarksideJ** [comment]
It was a nice idea @keveleigh to possibly allow users to create their own custom actions, sadly it was a solution in need of a problem, hence the rollback.

We may reevaluate in the future but for the foreseeable future, there is no demand.

**Comment by StephenHodgson** [comment]
@keveleigh any update on this?

**Comment by keveleigh** [comment]
Thanks for the explanation @SimonDarksideJ!

**Pull request merged**